crates/rspack_binding_options/src/options/raw_module/mod.rs

```diff
@@ -586,10 +585,10 @@ impl From<AssetGeneratorDataUrlFnArgs> for RawAssetGeneratorDataUrlFnArgs {
 
 impl From<RawAssetGeneratorDataUrlWrapper> for AssetGeneratorDataUrl {
   fn from(value: RawAssetGeneratorDataUrlWrapper) -> Self {
-    let handle = Handle::current();
+    use pollster::block_on;
```

Comment:
support nested block which tokio block_on doesn't support, will be removed when refactor fn to async